### ISSUE TITLE
backdrop-filter property droppable causes dragged item to disappear

### ISSUE THREAD

**Issue by mrap** [atlassian/react-beautiful-dnd#1826]
# Expected behavior

Dragging item does not disappear when dragging.

### Actual behavior

Dragging item disappears when dragging.  If you drag high enough on the page, the item actually appears from the bottom of the page.

### Steps to reproduce

Add the `backdrop-filter` css property to any parent of `Draggable` within the `DragDropContext`.
Any filter applied seems to cause the dragged items to disappear

### What version of `React` are you using?

16.13.1

### What version of `react-beautiful-dnd` are you running?

13.0.0

### What browser are you using?

Google Chrome Canary

### Demo

[DEMO](https://gfycat.com/graciouslegitimatebernesemountaindog.gif)

**Comment by 295** [comment]
A temporary solution:

```jsx
<Parent
  position="relative"
  _before={{
    content: '""',
    backdropFilter: "...",
    position: "absolute",
    width: "100%",
    height: "100%"
  }}
>
  <DragDropContext .../>
</Parent>
```

**Comment by rendomnet** [comment]
Is this bug will be fixed?

**Comment by ricosandyca** [comment]
I'm using Chakra UI portal while the card's dragging, and it's working fine.

https://user-images.githubusercontent.com/43330562/126056452-d8413543-b630-47ca-a697-1ab4a06e7dbd.mp4

**Comment by Jaagrav** [comment]
> I'm using Chakra UI portal while the card's dragging, and it's working fine.
> 
> ```js
> <Draggable ...>
>   {(provided, snapshot) =>
>     snapshot.isDragging ? (
>       // render your card in portal
>       <Portal>
>         <YourCard .../>
>       </Portal>
>     ) : (
>       <YourCard .../>
>     )
>   }
> </Draggable>
> ```
> 
>  Record_select-area_20210718120458.mp4

So how does ChakraUI apply backdrop-filter? Is it on the `::before`?

**Comment by ricosandyca** [comment]
> So how does ChakraUI apply backdrop-filter? Is it on the `::before`?

No, it applies the `backdrop-filter` directly on the component it self.
Portal is used to transport its wrapped component to the end of `document.body`. In other words, you can make your own portal to transport your component while it's getting dragged

**Comment by P4tr0ll** [comment]
I just encountered this bug and it was a pain to debug what was causing it.
backdrop-filter on one of the parent elements made the dragged element incorrectly offsetted

React - 18.2
Chrome - 121
React DnD - 13.1.8
